contracts/Account.sol

```diff
@@ -225,6 +226,14 @@ contract Account is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed, I
         }
     }
 
+    /**
+     * @notice Sets that address permissioned to pause/unpause this contract.
+     * @param _pauser The address that can pause/unpause this contract.
+     */
+    function setPauser(address _pauser) external onlyOwner {
```

Comment:
Should there be a hardcoded requirement that the pauser address of this contract is the same as the pauser of the multisig?

If the pauser addresses are different between multisig and other contracts, the TX will fail when the multisig `pauseContracts()` is called.

Same goes for all other contracts

Reply:
I don't hate the idea in principle, but not sure it's necessary? It's going to add a bit of complexity, and reduce modularity (e.g. if in the future we want several different pausers with different authentication) to avoid an unlikely configuration error that is fairly easy to remedy. How strongly do you feel about this?

Reply:
It's something I feel would be useful to avoid misconfiguring, and having to wait 7 days to reconfigure if we were to notice an issue when we need to pause.

It would bring peace of mind knowing that there is a hardcoded check.

In terms of complexity, this could be done by adding a modifier that checks if `_pauser == owner.pauser()`. WDYT?

Reply:
Implemented this as `setPauser` always setting the pauser to the owner.